src/app/contacts/contacts.component.scss

```diff
@@ -1,6 +1,7 @@
 
 .container {
-  font-family: "Roboto-Regular", sans-serif;
+  font-family: "Roboto", sans-serif;
```

Comment:
https://github.com/geekhub-php/theatre-front/blob/develop/src/assets/_variables.scss#L4

Reply:
Fixed.
Imported variables and useed some for container